low-level/files/opt/seagate/sspl/setup/sspl_dev_deploy

```diff
@@ -293,8 +293,8 @@ class SSPLDevDeploy:
 
         commands = [
             "%s post_install --config yaml://%s" % (setup_script, self.sample_global_config),
```

Comment:
at line 233, can you please change "srvnode1" to "srvnode-1". In provisoner config file they have define it as "srvnode-1".

Reply:
Earlier we used "srvnode-1" only. But you have told me to use "srvnode1". So, I have to revert this thing.

Reply:
I will do.